src/pages/Login.tsx

```diff
@@ -1,14 +1,50 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import { LinkFacade } from '../lib';
+import { Container } from '../components/Container';
+import { REACT_APP_API_BASE_URL, REACT_APP_GOOGLE_CLIENT_ID } from '../data';
+import { useGoogleAuth } from '../hooks';
+
+declare global {
+  interface Window {
+    google: typeof import('google-one-tap');
+  }
+}
 
 export function Login() {
+  const { handleGoogleLogin, error, loading } = useGoogleAuth();
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (window.google) {
+      window?.google.accounts.id.initialize({
+        client_id: REACT_APP_GOOGLE_CLIENT_ID as string,
+        callback: handleGoogleLogin,
+      });
+
+      window.google.accounts.id.renderButton(
+        document.getElementById('loginDiv') as HTMLDivElement,
+        {
+          theme: 'outline',
+          text: 'continue_with',
+          shape: 'pill',
+          type: 'standard',
+          logo_alignment: 'center',
+        },
+      );
+    }
+  }, [dispatch, handleGoogleLogin]);
```

Comment:
@SergeInGithub @musllim @shyakadavis 
Why is `dispatch` under the dependency array yet it is not used in the hook. 
```suggestion
  }, [handleGoogleLogin]);
```